### ISSUE TITLE
Only first two lines of the display are used

### ISSUE THREAD

**Issue by MarSoft** [dasburninator/klipper_lcd_menu#2]
Looks like the `__voron_display` display_data group only uses the first two rows of the 4-row display.
Strange; it declares 4 rows, but only first 2 are displayed in fact. Maybe a problem with my configuration? But all 4 rows are shown by default (display is mini12864).

**Comment by kwjett** [comment]
Do you have an enclosure temp sensor installed?
It should look like this:
![IMG_5279](https://github.com/Nighthawk3D/klipper_lcd_menu/assets/113549936/68b5d96d-8877-4530-ad87-c2fef4612684)

**Comment by dasburninator** [comment]
Do you have an enclosure temperature sensor? It needs to be named "chamber"

```
[temperature_sensor chamber]
sensor_type: Generic 3950
sensor_pin: PA1
min_temp: 0
max_temp: 100
gcode_id: C
```

It should look like this:
![IMG_5279](https://github.com/Nighthawk3D/klipper_lcd_menu/assets/41883401/4c4f6a37-2e92-4bb7-a7ea-755d0394226e)

**Comment by MarSoft** [comment]
No, I don't have an enclosure sensor (or enclosure itself). Could it be the reason why all the remaining fields in the lower 2 lines are hidden?

**Comment by MarSoft** [comment]
Confirmed. The bug disappeared after adding this line to my config file:
```
[display_data __voron_display chamber]
text: ~chamber~ 0~degrees~
```

Looks like any exception raised when rendering display text will stop rendering the remaining display sections.

**Comment by dasburninator** [comment]
I have a fix for it that I will push to the repo soon. Not sure that setting it to 0 is the best option here or leaving that entire display section blank.

The fix will be an additional variable in the lcd_menu_settings.cfg file to control the chamber display section.

**Comment by dasburninator** [comment]
Optionally you could also just comment out this line in `lcd_menu_settings.cfg` and revert to the stock display layout where the chamber sensor isn't part of the layout.

```
[include ./lcd_menu/lcd_tweaks.cfg]
```

**Comment by dasburninator** [comment]
Resolved from this pull request.
https://github.com/dasburninator/klipper_lcd_menu/pull/3
